src/server/services/records.js

```diff
@@ -36,8 +37,8 @@ function extractRecords (buffer) {
   const subcategory = cover['Detailed Expenditure Category']
 
   const records = [
-    { type: 'certification', content: certification },
-    { type: 'cover', content: cover }
+    { type: 'certification', upload, content: certification },
+    { type: 'cover', upload, content: cover }
```

Comment:
hmm, do these types of record not have a `subcategory` field?

Reply:
Hmm, that's true.  I'm not sure we actually use these records for anything other than extracting the subcategory at the moment.  Maybe we should just omit them?